src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourseActivityPayRate.kt

```diff
@@ -67,6 +68,10 @@ data class CourseActivityPayRate(
   override fun hashCode(): Int = javaClass.hashCode()
   override fun toString(): String =
     "CourseActivityPayRate(courseActivity=$courseActivity, iepLevel=$iepLevelCode, payBandCode=$payBandCode, startDate=$startDate)"
+
+  companion object {
+    fun preciseHalfDayRate(halfDayRate: BigDecimal) = halfDayRate.setScale(3, RoundingMode.HALF_UP)
+  }
```

Comment:
I had a test failure around BigDecimal precision and decided we should be more precise. Not sure it's really necessary though as we deal with small numbers.